renovate.json

```diff
@@ -1,40 +1,72 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "automerge": false,
-  "baseBranches": ["main"],
+  "baseBranches": ["main", "20.0.x"],
   "enabledManagers": ["npm", "bazel", "github-actions"],
   "stopUpdatingLabel": "action: merge",
-  "labels": ["target: patch"],
+  "labels": ["action: review"],
   "dependencyDashboard": true,
   "rangeStrategy": "pin",
```

Comment:
Note: I think we want to keep this here as pinning is preferred, and we generally don't use ranges for deps